PowerEditor/installer/nppSetup.nsi

```diff
@@ -187,8 +187,15 @@ updaterDone:
 		StrCpy $LANGUAGE "$tempLng" ; set default language
 	${EndIf}
 	
+	Call unsupportedLanguageToEnglish

```

Comment:
That's PR #14125's which didn't work - why should we apply it again?

Reply:
I have already [explained](https://github.com/notepad-plus-plus/notepad-plus-plus/issues/3844#issuecomment-1736463273) this, I am copying here my previous note:

"@mpheath has already correctly described most of the problems and possible solutions. But his PR is not working because while he fixed the possibly wrong regkey/$LANGUAGE value before invoking that NSIS MUI_LANGDLL_DISPLAY at .onInit, if NSIS own lang-detection method of that MUI_LANGDLL_DISPLAY fails before displaying the lang-selection dlg, NSIS engine reaches rather for the statically predefined first member of the N++ lang-list than for the already correctly preset $LANGUAGE var. And then it updates the $LANGUAGE accordingly (currently either with that Afrikaans or with the complete nonsense value "Installer Language""

Now I am using his `unsupportedLanguageToEnglish` func because of [you have asked](https://github.com/notepad-plus-plus/notepad-plus-plus/pull/14215#discussion_r1349532697) for such more universal detection method...